extension/json/src/functions/export_functions/json_export.cpp

```diff
@@ -32,7 +32,8 @@ struct ExportJSONSharedState : public ExportFuncSharedState {
     std::unique_ptr<FileInfo> fileInfo;
     std::vector<std::string> jsonValues;
 
-    void init(main::ClientContext& context, const ExportFuncBindData& bindData) override {
+    void init(main::ClientContext& context, const ExportFuncBindData& bindData,
+        const std::shared_ptr<std::atomic<bool>>& /*parallelFlag*/) override {
```

Comment:
you don't have to wrap shared_ptr with reference. just pass std::atomic<bool>& there.